### PR TITLE
PHP CS Fixer & Prettier

### PR DESCRIPTION
**Pull request by driesvints**

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        'declare_equal_normalize' => ['space' => 'none'],
+        'function_typehint_space' => true,
+        'new_with_braces' => true,
+        'method_argument_space' => ['on_multiline' => 'ensure_fully_multiline'],
+        'no_empty_statement' => true,
+        'no_leading_import_slash' => true,
+        'no_leading_namespace_whitespace' => true,
+        'no_whitespace_in_blank_line' => true,
+        'return_type_declaration' => ['space_before' => 'none'],
+        'single_trait_insert_per_statement' => true,
```
Comment (joedixon) [inline .php_cs.dist:32]:
```suggestion
        'single_trait_insert_per_statement' => true,
        'no_extra_blank_lines' => [
            'curly_brace_block',
            'extra',
            'parenthesis_brace_block',
            'throw',
            'use',
        ],
        'ternary_operator_spaces' => true,
        'not_operator_with_successor_space' => true,
        'single_blank_line_before_namespace' => true,
```

Just did a comparison against my config and I have these additional options which may be useful?

Comment (driesvints) [reply .php_cs.dist:42]: Done, thanks!

**Comment by bramus** [comment]
Is there a specific reason to have one job extract the branch name using `echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"` and have the other job use `github.head_ref`?

**Comment by driesvints** [comment]
@bramus good spot actually. I just copied the workflows from the article I read 😬

**Pull request merged**